include/sundials/sundials_linearsolver.h

```diff
@@ -226,18 +226,6 @@ SUNErrCode SUNLinSolFree(SUNLinearSolver S);
 #define SUNLS_QRFACT_FAIL      807 /* QRfact found singular matrix  */
 #define SUNLS_LUFACT_FAIL      808 /* LUfact found singular matrix  */
 
-/* -----------------------------------------------------------------------------
- * SUNLinearSolver messages
- * ---------------------------------------------------------------------------*/
-
-#if defined(SUNDIALS_EXTENDED_PRECISION)
-#define SUNLS_MSG_RESIDUAL "\t\tlin. iteration %ld, lin. residual: %Lg\n"
-#elif defined(SUNDIALS_DOUBLE_PRECISION)
-#define SUNLS_MSG_RESIDUAL "\t\tlin. iteration %ld, lin. residual: %g\n"
-#else
-#define SUNLS_MSG_RESIDUAL "\t\tlin. iteration %ld, lin. residual: %g\n"
-#endif
```

Comment:
Were these unused? I'm guessing they might have been left over the old monitoring output before we moved to logging.

Reply:
> Were these unused?

Yes. I have another branch with extra warnings enabled to catch these: https://github.com/LLNL/sundials/tree/feature/unused-macro. Once this PR is merged, I'll make another for that